operations/network/NMN_Isolation.md

```diff
@@ -0,0 +1,243 @@
+# Node Management Network (NMN) Isolation
+
+## NMN Isolation Overview
+
+NMN Isolation on the management network limits traffic on the NMN to only types and directions required for the operation of CSM and user workloads.
+**NMN Isolation is only available on systems with Aruba switches**. The feature consists of three main sub-features:
+
+- ACLs allowing access to only required CSM services from managed nodes (compute, UAN, and so on)
+- ACLs preventing Mountain compute cabinets (EX) from communicating with each other
+- PVLAN to prevent River managed nodes (compute, UAN, and so on) from communicating with each other
+
+NMN Isolation alleviates the need for most host-based firewalls on management nodes (managers, workers, and storage NCN).
+The PVLAN sub-feature also removes the need for host-based firewalls on UAN (and other River managed nodes) by limiting access over the NMN.
+
+NMN Isolation is available in the following commands of CANU via the `--enable-nmn-isolation --enable-pvlan` options:
+
+- `canu generate switch config` generates a configuration file for a _single switch_ on the management network
+- `canu generate network config` generates all configuration files for _all switches_ on the management network
+- `canu validate switch config` compares the running configuration of a _single switch_ with a configuration generated by CANU - a configuration diff
+- `canu validate network config` provides a summary comparison between generated and running configurations for _all switches_ in the management network
+- `canu test` runs a full diagnostic on all management network switches, including tests for NMN Isolation ACLs
+
+Details of allowed services and the network changes involved in the NMN Isolation feature can be found in [NMN Isolation Details](#nmn-isolation-details).
+
+## Enabling NMN Isolation
+
+A network outage window is required to configure NMN Isolation on a system.  NMN Isolation changes are decoupled from other changes to CSM and the network outage window could be prior or after the CSM upgrade.
+However, if IPv6 is being enabled as part of a system upgrade, enabling NMN Isolation at the same time as IPv6, and prior to the CSM upgrade is required to avoid multiple network outage windows.
+
+### 1. Preparation
+
+All preparation steps can be performed prior to an established window.  Preparation steps do not change the running network configurations.
+
+1. (`ncn-m#`) Gather a list of switches.
+
+    - For an upgrade the list of switches can be found from the command `grep sw- /etc/hosts`
+    - For a fresh installation the switches can be found in the latest system SHCD spreadsheet
+
+2. Determine if the system is a TDS or Full system to be used in CANU commands.
+
+    - If there are leaf switches the system is `Full`
+    - Otherwise, the system is a `TDS`
+
+3. (`ncn-m#`) Ensure the CANU version is `2.0.2` or greater, otherwise [upgrade to the latest version](management_network/canu_install_update.md)
```

Comment:
```suggestion
1. (`ncn-m#`) Ensure the CANU version is `2.0.2` or greater, otherwise [upgrade to the latest version](management_network/canu_install_update.md)
```